cpp/models/ide_secir/parameters.h

```diff
@@ -270,9 +270,9 @@ class Parameters : public ParametersBase
             return true;
         }
 
-        for (size_t i = 0; i < (int)InfectionTransition::Count; i++) {
+        for (size_t i = 1; i < (int)InfectionTransition::Count; i++) {
```

Comment:
A comment would be helpful to understand why not all TransitionDistributions are checked.